plugin.yaml

```diff
@@ -542,7 +545,7 @@ node_types:
   # distributed logical routers interface ospf areas, use only after all
   # interfaces creation!
   cloudify.nsx.ospf_areas:
-    derived_from: cloudify.nodes.Port
+    derived_from: cloudify.nodes.ApplicationServer
```

Comment:
I would probably have these as cloudify.nodes.Root given the available type options... I'm not sure anything else fits.

(applies to all of the following derived_froms)